### PR TITLE
bug fix: corrected handling of ExceptionInfo

### PR DESCRIPTION
**Pull request by mt3593**

### CODE REVIEW

File: test/clj_http_hystrix/core_test.clj [hunk 1]
```diff
@@ -7,7 +7,7 @@
 
 (fact "hystrix wrapping with fallback"
       (http/get "http://httpstat.us/400" {:throw-exceptions false
-                                          :hystrix/fallback-fn (constantly "foo")})
```
Comment (joelittlejohn) [inline test/clj_http_hystrix/core_test.clj:10]: Any reason for this change?

**Comment by joelittlejohn** [comment]
Cool, looks like you have made the wrapper fully compatible with `:throw-exceptions` now.

File: src/clj_http_hystrix/core.clj [hunk 2]
```diff
@@ -15,12 +14,18 @@
...
-      (-> (ex-data e) :object)
-      {:status 500})))
+(defn default-fallback [req this]
+  (.getFailedExecutionException this))
+
+(defn handle-exception
+  [f req]
+  (let [response (f)]
+    (if (instance? clojure.lang.ExceptionInfo response)
+      (if (or (not (contains? req :throw-exceptions))
```
Comment (joelittlejohn) [inline src/clj_http_hystrix/core.clj]: `(:throw-exceptions req true)` ?

File: test/clj_http_hystrix/core_test.clj (hunk 1 above)
Comment (mt3593) [inline test/clj_http_hystrix/core_test.clj:10]: was playing about with another approach and forgot to put it back!

File: src/clj_http_hystrix/core.clj (hunk 2 above)
Comment (mt3593) [inline src/clj_http_hystrix/core.clj]: forgot about the default value! dam

**Comment by joelittlejohn** [comment]
I've added you as a committer, feel free to merge this. It would be good to add a test for the improved throw-exceptions support.

**Pull request closed without merging**